iOS/Flows/Conversation/Cells/ConversationMessageCell.swift

```diff
@@ -18,9 +18,9 @@ class ConversationMessageCell: UICollectionViewCell {
     var handleTappedMessage: ((Messageable) -> Void)?
     var handleDeleteMessage: ((Messageable) -> Void)?
 
-    private let collectionView: UICollectionView = {
-        let layout = ConversationMessageCellLayout()
-        let cv = UICollectionView(frame: .zero, collectionViewLayout: layout)
+    private let collectionLayout = ConversationMessageCellLayout()
```

Comment:
The cell layout needs to have a message controller assigned to it so I'm making it a property.